Helpers/Eligibility.php

```diff
@@ -134,10 +121,16 @@ public function __construct(
      * @throws LocalizedException
      * @throws NoSuchEntityException
      * @throws RequestError
-     * @throws InvalidArgumentException
+     * @throws \InvalidArgumentException
      */
     private function getPlansEligibility(): array
     {
+        try {
+            $quote = $this->getEligibilityQuote();
+        } catch (\InvalidArgumentException $e) {
+            throw new \InvalidArgumentException($e->getMessage());
```

Comment:
don't use FQDN

add import with `\InvalidArgumentException`

Reply:
you should use `use \InvalidArgumentException;` in the header of your class file.

then, you can after call the `InvalidArgumentException` without FQDN (without `\` suffix)